raiden/transfer/mediated_transfer/events.py

```diff
@@ -339,7 +339,7 @@ def __init__(
             channel_identifier: ChannelID,
             message_identifier: MessageID,
             payment_identifier: PaymentID,
-            amount: TokenAmount,
```

Comment:
The `SecretRequest` contains an amount of tokens to unlock for the transfer. I think we use `PaymentAmount` for end to end payments and `TokenAmount` for anything else.

But in fact this is not well standardized across the codebase and should be discussed.

Reply:
* I'll add a comment about `PaymentAmount` `#  for end-to-end payments`
* I'll try to use `TokenAmount` here.